src/utils/filter-utils.js

```diff
@@ -99,6 +102,26 @@ export const TIME_ANIMATION_SPEED = [
   }
 ];
 
+export function generatePolygonFilter(layer, feature) {
+  return generateFilter({
+    dataId: layer.config.dataId,
+    layerId: layer.id,
+    fixedDomain: false,
```

Comment:
I think Polygon filter should be `fixedDomain: true`, like time filter. It's weird when people select filter to a subset of point and its color changes.